client/src/components/admin/AdminDashboard.tsx

```
@@ -0,0 +1,123 @@
+        ${sidebarOpen ? 'translate-x-0' : '-translate-x-full md:translate-x-0'}`}
+      >
+        <div className="p-4 border-b text-xl font-bold text-center text-indigo-900">
+          Admin Dashboard
+        </div>
+        <nav className="flex flex-col p-4 space-y-4">
+          <button
+            onClick={() => handleSectionChange('overview')}
+            className={`flex items-center gap-3 p-3 rounded-md text-gray-800 ${
+              activeSection === 'overview' ? 'bg-purple-100 text-purple-700' : 'hover:bg-purple-50'
+            }`}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)